demo-circuit/src/naive_threshold_sig/mod.rs

```diff
@@ -267,56 +291,57 @@ pub fn generate_parameters(max_pks: usize) -> Result<Parameters<MNT4>, Synthesis
         _field:                   PhantomData
     };
 
-    let params = generate_random_parameters::<MNT4, _, _>(c, &mut rng);
-    params
+    let universal_srs = MarlinInst::universal_setup(2usize.pow(20), 2usize.pow(20), 2usize.pow(20), &mut rng).unwrap();
```

Comment:
If you want to give exact numbers here and not waste too much time in the generation procedure (you know that we are going to remove it by passing the segment size), the Marlin::index() function accepts only the circuit and in that function there are a couple of prints you can use to get exact data about the number of constraints, number of variables, and number of non zero